logic/models.py

```diff
@@ -11,19 +11,28 @@ class Bullet(Object):
     angle: float
     velocity: float
     damage: float
+    creationTime: float
+    rect: tuple
 
     def update(self, deltatime: float):
         dir = get_direction(self.angle) * self.velocity * deltatime
         self.posX += dir.x
-        self.posY += dir.y        
+        self.posY += dir.y
+
+        
+        if ((pygame.time.get_ticks() - self.creationTime) / 1000 > 5):
```

Comment:
If i am not mistaken the number 5 represents the life time of a bullet. Not preferred to have numbers hard coded in the code. Best way is to allocate a property for that number in Bullet class